addons/website_sale/templates/snippets/s_add_to_cart.xml

```diff
@@ -7,7 +7,12 @@
             tracking utilities will be fully integrated into a service.
         -->
         <div class="s_add_to_cart oe_website_sale" data-action="add_to_cart">
-            <button class="s_add_to_cart_btn disabled btn btn-secondary mb-2" data-action="add_to_cart">
+            <!-- TODO(loti): data-show-quantity is never set -->
```

Comment:
TODO to explanation instead, since we cannot do it right now (will be for when we move the show-quantity in the session info)